CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Clarify documentation for --scan-host parameter [#1277](https://github.com/greenbone/gvmd/pull/1277)
 - In result iterator access severity directly if possible [#1321](https://github.com/greenbone/gvmd/pull/1321)
 - Change SCAP and CERT data to use new severity scoring [#1333](https://github.com/greenbone/gvmd/pull/1333) [#1357](https://github.com/greenbone/gvmd/pull/1357) [#1365](https://github.com/greenbone/gvmd/pull/1365)
+- The internal list of current Local Security Checks for the Auto-FP feature was updated [#1381](https://github.com/greenbone/gvmd/pull/1381)
```

Comment:
```suggestion
- The internal list of current Local Security Checks for the 'Auto-FP' and 'Closed CVEs' features was updated [#1381](https://github.com/greenbone/gvmd/pull/1381)
```

AFAIK the Auto-FP feature will be gone in 21.04 so not sure if it worth to be mentioned here. In addition this PR probably should rebased against the 21.04 branch.

Reply:
> should rebased against the 21.04 branch.
You mean 20.08 right?

Reply:
No, 21.04 was / is correct. To the best of my knowledge Notus will first appears in 21.04 so there is no point in updating that list in 20.08 as it won't be used / give any results of that family in 20.08.

Reply:
I think this was a misunderstanding, because we both thought there was a 21.04 branch here. So the master branch is the correct branch as a target, right? @mattmundell

Reply:
yes, master is the place for 21.04 work.